cdap-docs/_common/_themes/cdap/layout.html

```diff
@@ -117,11 +117,6 @@
     })(window,document,'script','dataLayer','{{ html_google_tag_manager_code }}');</script>
     {%- endif %}
     <!-- Google Tag Manager end -->
-    
-    <!-- Amazon SQS Page Usage Logging start -->
-    <script type="text/javascript" src="https://sdk.amazonaws.com/js/aws-sdk-2.1.35.min.js"></script>
-    <script type="text/javascript" src="http://docs.cask.co/resources/scripts/aws-sqs-1.0.0.js"></script>
```

Comment:
I see, because the existing documentation references that file.
Makes sense.
LGTM.